scripts/AssertPrGreenReviewHeuristics.ps1

```
@@ -105,6 +105,31 @@ function Test-StaleBotReviewCanBeIgnored {
+    if ($author -notmatch '^claude(?:\[bot\])?$' -or $HeadSha -notmatch '^[0-9a-f]{40}$') {
+        return $false
+    }
+
+    $markers = [regex]::Matches(
+        [string]$Review.body,
+        '(?im)^\s*<!--\s*REVIEW_VERDICT:\s*(CLEAR|BLOCKING)\s+HEAD:\s*([0-9a-f]{40})\s*-->\s*$'
+    )
+    if ($markers.Count -ne 1 -or
+        $markers[0].Groups[1].Value -ne 'CLEAR' -or
+        $markers[0].Groups[2].Value -ne $HeadSha) {
+        return $false
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Test-TrustedBotClearVerdict' returns an object of type 'System.Boolean' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)